### PR TITLE
Rm99 migration

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
I have opened a new PR because I deleted the previous PR due to a bug. If this PR is acceptable, could it be merged after the crash is fixed?

### CODE REVIEW

**Comment by BorkBorkGoesTheCode** [comment]
@Rivet-the-Zombie this is the working version. I moved the RM99 to pistol_obscure.

File: data/json/items/gun/8x40mm.json [hunk 1]
```diff
@@ -204,4 +204,236 @@
     "magazine_well" : 2,
     "flags": [ "WATERPROOF_GUN", "NEVER_JAMS" ]
   }
+[
```
Comment (Rivet-the-Zombie) [inline data/json/items/gun/8x40mm.json]: An odd separator here is breaking the .json parser. Also, what's with all the duplicated guns listed in here?

**Comment by BorkBorkGoesTheCode** [comment]
Sorry, double copypaste. I have fixed it now.

**Comment by Rivet-the-Zombie** [comment]
And accidentally introduced a bunch of merge conflicts.

**Comment by BorkBorkGoesTheCode** [comment]
What causes those merge conflicts?

**Comment by Rivet-the-Zombie** [comment]
Probably all the random whitespace alterations that got mixed in there.

**Comment by BorkBorkGoesTheCode** [comment]
I have fixed the conflicts

**Comment by BorkBorkGoesTheCode** [comment]
@Rivet-the-Zombie are you still online?

**Comment by Rivet-the-Zombie** [comment]
Kinda sorta; when I remember to check my phone.

**Comment by BorkBorkGoesTheCode** [comment]
Thank you.

**Pull request merged**